<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Color_Widget</class>
 <widget class="QWidget" name="Form_Color_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>96</width>
    <height>30</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>45</width>
    <height>30</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>96</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>1</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QToolButton" name="tb_action">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton
{ border-style:none;}

QToolButton:hover
{
background:#e5e5e5;
}</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_color">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>4</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>4</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="tb_menu">
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QToolButton:hover
{
background:#e5e5e5;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="textedit.qrc">
       <normaloff>:/res/icon/dropdown.png</normaloff>:/res/icon/dropdown.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="textedit.qrc"/>
 </resources>
 <connections/>
</ui>
